src/local/butler/lint.py

```diff
@@ -34,23 +164,32 @@ def execute(_):
   else:
     _, output = common.execute('git diff --name-only FETCH_HEAD')
 
+  file_paths = [f for f in output.splitlines() if os.path.exists(f)]
```

Comment:
If file is deleted, then we don't want lint to run on it and cause exception.

Reply:
What changed though?

Reply:
We had the "if os.path.exists(file_path):" repeated inside every loop. Now we just do it once at start instead of repeating it for every file type check (like before it was repeated for python, go files, etc).